.github/workflows/ci.yaml

```diff
@@ -0,0 +1,13 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  validate:
+    runs-on: ubuntu-20.04
```

Comment:
This is an image right? Downloaded from dockerhub?

Reply:
They actually call them "GitHub-hosted runners" https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners and looks like they live here https://github.com/actions/virtual-environments

Reply:
Interesting. Thanks.